seed/csharp-sdk/audiences/src/SeedAudiences/Foo/FooClient.cs

```diff
@@ -15,7 +15,7 @@ public FooClient(RawClient client)
         _client = client;
     }
 
-    public async Task<ImportingType> FindAsync(FindRequest request)
+    public async Task<ImportingType> FindAsync(FindRequest request, RequestOptions? options)
```

Comment:
i think we wanna do `RequestOptions? options = null` so that it can be omitted completely

Reply:
I think we already get that without the `= null` since it's specified as an optional (via `?`) - I wrote up a test case to confirm.

Reply:
I stand corrected - we should add that in here!